packages/grafana-ui/src/components/Tags/Tag.mdx

```diff
@@ -0,0 +1,19 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import { Tag } from './Tag';
+
+<Meta title="MDX|Tag" component={Tag} />
```

Comment:
Meta is not needed since the metadata is defined in the story file, if I remember correctly.

Reply:
Hmm, I didn't add any metadata to the story file. Do I need to?

Reply:
No, it's just the title in this case. If you remove it, you'll probably see that it works just as before.

Reply:
It's possible to add parameters and such in the `<Meta/>` component, but that's mainly for MDX files without any connection to a story.